dist/index.html

```diff
@@ -10,9 +10,9 @@
     <header class="header">
       <nav class="nav">
         <a href="#"><img class="logo" alt="" /></a>
-        <a href="#" class="nav-link beef">Beef</a>
-        <a href="" class="nav-link">Chicken</a>
-        <a href="" class="nav-link">Seafood</a>
+        <li class="nav-link beef">Beef</li>
+        <li class="nav-link">Chicken</li>
+        <li class="nav-link">Seafood</li>
```

Comment:
Great work so far, 
Kindly wrap the 'li' tags in a 'ul' or 'ol' tag to increase accessibility as Screen readers notify users when they come to a list, and tell them how many items are in a list.